src/action.js

```diff
@@ -41,6 +45,7 @@ module.exports = async function run({
     const { pull_request } = context.payload
 
     if (!pull_request && !PR_NUMBER) {
+      core.setOutput('merge_status', MERGE_STATUS.skippedNotADependabotPr)
```

Comment:
let's extract the string "merge_status" in a constant as well to avoid the multiple repetitions you have in code now